src/ucp/api/ucp.h

```diff
@@ -770,20 +771,20 @@ typedef struct ucp_worker_listener_params {
     /**
      * An address in the form of a sockaddr.
      * This field is mandatory for filling (along with its corresponding bit
-     * in the field_mask - @ref UCP_WORKER_LISTENER_PARAM_FIELD_SOCK_ADDR).
-     * The @ref ucp_worker_listen routine will return with an error if sockaddr
+     * in the field_mask - @ref UCP_LISTENER_PARAM_FIELD_SOCK_ADDR).
+     * The @ref ucp_listener_create routine will return with an error if sockaddr
      * is not specified.
      */
     ucs_sock_addr_t                sockaddr;
 
     /**
      * Handler to endpoint creation in a client-server connection flow.
      * In order for the callback inside this handler to be invoked, the
-     * UCP_WORKER_LISTENER_PARAM_FIELD_CALLBACK needs to be set in the
+     * UCP_LISTENER_PARAM_FIELD_CALLBACK needs to be set in the
```

Comment:
This should be UCP_LISTENER_PARAM_FIELD_ACCEPT_HANDLER. Also, it should have  `@ref``.